commands/audit/scarunner.go

```diff
@@ -75,6 +76,11 @@ func buildDepTreeAndRunScaScan(auditParallelRunner *utils.SecurityParallelRunner
 		// Get the dependency tree for the technology in the working directory.
 		treeResult, bdtErr := buildDependencyTree(scan, auditParams)
 		if bdtErr != nil {
+			var installForbiddenErr *biutils.ErrInstallForbidden
```

Comment:
`ErrInstallForbidden` is a bit confusing, I would change it to `ErrProjectNotInstalled`